tests/fitting/common/test_regularization.py

```diff
@@ -193,3 +215,46 @@ def arrays_allclose(array1, array2):
     assert inside_result_3D
     # Point point_3D_outside outside region:
     assert not outside_result_3D
+
+
+def test_regularization_for_three_element_system(test_dir, memory_jobstore, clean_dir):
+    """Tests auxiliary functions related to three_element_system."""
+    from ase.io import read
+
+    file = test_dir / "fitting" / "ref_files" / "test_regularization.extxyz"
+    atoms = read(file, ":")
+
+    points = label_stoichiometry_volume(
+        atoms,
+        isolated_atom_energies={3: 0.1, 15: 0.2, 16: 0.3},
+        energy_name="energy",
+        element_order=[3, 15, 16],
+    )
+
+    hull = calculate_hull_nd(points)
+
+    des = [
+        get_e_distance_to_hull_nd(
+            hull,
+            at,
+            energy_name="energy",
+            isolated_atom_energies={3: 0.1, 15: 0.2, 16: 0.3},
+            element_order=[3, 15, 16],
+        )
+        for at in atoms
+    ]
+
+    assert all(d >= -1e-6 for d in des)
```

Comment:
```suggestion
    assert np.all(np.array(des) >= -1e6)
```

Replacing with this line should fix this failing test